projects/1.16-fabric/assets/dungeons-mod-lite/dungeonsmod/lang/zh_cn.json

```
@@ -0,0 +1,190 @@
+	"item.dungeonsmod.lordskeleton_egg": "骷髅领主刷怪蛋",
+	"item.dungeonsmod.deadhound_egg": "死亡猎犬刷怪蛋",
+	"item.dungeonsmod.crow_egg": "乌鸦刷怪蛋",
+	"item.dungeonsmod.notehead_egg": "音符兽刷怪蛋",
+	
+	"item.dungeonsmod.winterhunter_egg": "凛冬猎人刷怪蛋",
+	"item.dungeonsmod.pice_egg": "皮切刷怪蛋",
+	"item.dungeonsmod.leech_egg": "水蛭刷怪蛋",
+	"item.dungeonsmod.pitcher_egg": "陶罐怪刷怪蛋",
+	"item.dungeonsmod.whirlwind_egg": "旋风刷怪蛋",
+	
+	"item.dungeonsmod.crawler_egg": "生化大蛇刷怪蛋",
+	"item.dungeonsmod.deserted_egg": "遗忘者刷怪蛋",
+	"item.dungeonsmod.ironslime_egg": "钢铁史莱姆刷怪蛋",
+	"item.dungeonsmod.king_egg": "国王刷怪蛋",
+	"item.dungeonsmod.kraken_egg": "克拉肯刷怪蛋",
+	"item.dungeonsmod.sun_egg": "日之子刷怪蛋",
+	"item.dungeonsmod.voidmaster_egg": "虚空领主刷怪蛋",
+	
+	"item.dungeonsmod.guard_egg": "皇家守卫刷怪蛋",
+	"item.dungeonsmod.scientist_egg": "疯狂科学家刷怪蛋",
+	"item.dungeonsmod.slimewolf_egg": "史莱姆狼刷怪蛋",
+	
+	"filled_map.dungeonsmod:the_house": "宅邸",
+	"filled_map.dungeonsmod:the_mines": "钢铁矿洞",
+	"filled_map.dungeonsmod:the_ship": "沉船",
+	"filled_map.dungeonsmod:the_castle": "城堡",
+	"filled_map.dungeonsmod:the_lab": "实验室",
+	
+	"entity.dungeonsmod.extrapart": "弗波",
+	
+	"entity.dungeonsmod.anthermite": "安赛蜜火蚁",
+	"entity.dungeonsmod.mimic": "模仿者",
+	"entity.dungeonsmod.piranha": "食人鱼",
+	"entity.dungeonsmod.rogue": "恶棍",
+	"entity.dungeonsmod.slimond": "钻石史莱姆",
+	
+	"entity.dungeonsmod.haunter": "狩猎者",
+	"entity.dungeonsmod.lordskeleton": "骷髅领主",
+	"entity.dungeonsmod.deadhound": "死亡猎犬",
+	"entity.dungeonsmod.crow": "乌鸦",
+	"entity.dungeonsmod.notehead": "音符兽",
+	
+	"entity.dungeonsmod.winterhunter": "凛冬猎人",
+	"entity.dungeonsmod.pice": "皮切",
+	"entity.dungeonsmod.leech": "水蛭",
+	"entity.dungeonsmod.pitcher": "陶罐怪",
+	"entity.dungeonsmod.whirlwind": "旋风",
+	
+	"entity.dungeonsmod.crawler": "生化大蛇",
+	"entity.dungeonsmod.deserted": "遗忘者",
+	"entity.dungeonsmod.ironslime": "钢铁史莱姆",
+	"entity.dungeonsmod.king": "国王",
+	"entity.dungeonsmod.kraken": "克拉肯",
+	"entity.dungeonsmod.sun": "日之子",
+	"entity.dungeonsmod.voidmaster": "虚空领主",
+	
+	"entity.dungeonsmod.guard": "皇家守卫",
+	"entity.dungeonsmod.scientist": "疯狂科学家",
+	"entity.dungeonsmod.slimewolf": "史莱姆狼",
+	
+	"entity.minecraft.villager.traveler": "旅行者",
+	
+	"item.dungeonsmod.back_pergamine": "返回符咒",
+	"enchantment.level.32": "XXXII",
+	"enchantment.level.20": "XX",
+	"enchantment.level.17": "XVII",
+	"filled_map.village": "村庄",
+	"item.dungeonsmod.home_gem": "炉石",
+	"item.dungeonsmod.home_gem.tooltip_1": "链接",
+	"item.dungeonsmod.sun_overdrive": "太阳驱动器",
+	"item.dungeonsmod.cooldown": "暂停时间：%d 分钟 %d 秒",
+	"item.dungeonsmod.soul_bottle": "灵魂瓶",
+	"item.dungeonsmod.golden_bone": "金色骨头",
+	"item.dungeonsmod.piranha_bucket": "食人鱼桶",
+	"item.dungeonsmod.translatable_book": "多语言书籍",
+	
+	"item.dungeonsmod.winterhunter1.name": "凛冬猎手的弓",
+	"item.dungeonsmod.winterhunter1.lore": "在它的弦上，安放着一些野生动物的灵魂。",
+	
+	"item.dungeonsmod.deserted1.name": "古老的巫师之剑",
+	"item.dungeonsmod.deserted1.lore": "巫师的伟大力量在你手上。",
+	"item.dungeonsmod.ironslime1.name": "古老的被遗忘的法曼斯镐",
```

```suggestion
	"item.dungeonsmod.ironslime1.name": "古老的被遗忘的工头的镐",
```
原文`foremans`应该是少了个标点符号，即`foreman's`，`foreman`工头也符合钢铁史莱姆在矿井里发现的情况。